GoSSB/GoSSB.xcodeproj/project.pbxproj

```diff
@@ -235,7 +235,7 @@
 			);
 			runOnlyForDeploymentPostprocessing = 0;
 			shellPath = /bin/sh;
-			shellScript = "cp -r ${BUILD_DIR}/Products/GoSSB.xcframework/ ${PROJECT_DIR}/../Frameworks/GoSSB.xcframework\n\n";
+			shellScript = "rsync -t ${BUILD_DIR}/Products/GoSSB.xcframework/ ${PROJECT_DIR}/../Frameworks/GoSSB.xcframework\n\n";
```

Comment:
I learned recently that you shouldn't use `cp` in a build phase because it changes the file's last modified time and can cause unnecessary rebuilds. `rsync -t` preserves the last modified time.